plugins/contentful/personal_access_token.go

```diff
@@ -0,0 +1,81 @@
+package contentful
+
+import (
+	"context"
+	"encoding/json"
+
+	"github.com/1Password/shell-plugins/sdk"
+	"github.com/1Password/shell-plugins/sdk/importer"
+	"github.com/1Password/shell-plugins/sdk/provision"
+	"github.com/1Password/shell-plugins/sdk/schema"
+	"github.com/1Password/shell-plugins/sdk/schema/credname"
+	"github.com/1Password/shell-plugins/sdk/schema/fieldname"
+)
+
+func PersonalAccessToken() schema.CredentialType {
+	return schema.CredentialType{
+		Name:    credname.PersonalAccessToken,
+		DocsURL: sdk.URL("https://www.contentful.com/developers/docs/references/content-management-api/#/reference/personal-access-tokens"),
+		Fields: []schema.CredentialField{
+			{
+				Name:                fieldname.Token,
+				MarkdownDescription: "Personal Access Token used to authenticate to Contentful.",
+				Secret:              true,
+				Composition: &schema.ValueComposition{
+					Length: 49,
+					Prefix: "CFPAT-",
+					Charset: schema.Charset{
+						Uppercase: true,
+						Lowercase: true,
+						Digits:    true,
+					},
+				},
+			},
+		},
+		DefaultProvisioner: provision.TempFile(
+			contentfulConfig,
+			provision.AtFixedPath("~/.contentfulrc.json"),
+		),
+		Importer: importer.TryAll(
+			TryContentfulConfigFile(),
+		)}
```

Comment:
Let's use `TryContentfulConfigFile()` directly, no need for `TryAll` if there is only one importer.